src/Compilers/VisualBasic/Portable/Scanner/CharacterInfo.vb

```diff
@@ -488,4 +488,37 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Function
     End Class
 
+    Friend Module CharExts
+
+        <Runtime.CompilerServices.Extension>
+        Friend Function IsAnyOf(c As Char, c0 As Char, c1 As Char) As Boolean
+            Return (c = c0) OrElse (c = c1)
+        End Function
+
+        <Runtime.CompilerServices.Extension>
+        Friend Function IsAnyOf(c As Char, c0 As Char, c1 As Char, c2 As Char) As Boolean
+            Return (c = c0) OrElse (c = c1) OrElse (c = c2)
+        End Function
+
+        <Runtime.CompilerServices.Extension>
+        Friend Function IsAnyOf(c As Char, c0 As Char, c1 As Char, c2 As Char, c3 As Char) As Boolean
+            Return (c = c0) OrElse (c = c1) OrElse (c = c2) OrElse (c = c3)
+        End Function
+
+        <Runtime.CompilerServices.Extension>
+        Friend Function IsNoneOf(c As Char, c0 As Char, c1 As Char) As Boolean
```

Comment:
Consider changing the name to something more descriptive. For example, IsNotEqualToAll.